### ISSUE TITLE
Results on Hyperpartisian

### ISSUE THREAD

**Issue by amineabdaoui** [allenai/longformer#234]
Hi @ibeltagy

I am not able to reproduce the [paper](https://arxiv.org/pdf/2004.05150.pdf) results on the hyperpartisian dataset. 

I first tried evaluating on the real separate test set that can be found [here](https://zenodo.org/record/1489920#.YnI13tpBw2w), then I took 10% of the train as described on the [paper](https://arxiv.org/pdf/2004.05150.pdf).

I think that since this dataset is small the train / test split has an important impact on the results (sometimes Roberta on 512 tokens is doing better than longformer on 4096). 

Any suggestions?

Thanks

Ps. I am using the transformers' Trainer API with the same hyperparameter setting that you presented on the paper.

**Comment by armancohan** [comment]
The reprocessing and the exact data split to use will affect the results.
We have a preprocessing script for this here: https://github.com/allenai/longformer/blob/master/scripts/hp_preprocess.py
which includes steps to prepare and use this dataset.

Hope this helps.

**Comment by amineabdaoui** [comment]
I tried the above mentioned script to preprocess the data and generate the exact same splits. 
This script uses the [hp-splits.json](https://github.com/allenai/longformer/blob/master/scripts/hp-splits.json) file which contains the already separated document ids.
However, I found out that many documents are shared between the train and test splits. 
Actually, 51 out of the 65 test documents are also included in the train!

Here are the document ids that are common between the train and the test: 
`common_train_test = [517, 23, 459, 593, 258, 227, 204, 159, 214, 82, 564, 610, 21, 434, 625, 197, 202, 489, 404, 400, 551, 320, 36, 435, 183, 19, 253, 231, 383, 572, 201, 528, 15, 116, 265, 342, 465, 436, 442, 547, 282, 535, 256, 160, 140, 555, 51, 540, 165, 504, 181]`

The same issue was observed between the train and the dev and between the test and the dev. 

@armancohan, @ibeltagy do you confirm this flaw in the hyperpartisian evaluation or am I missing something?

**Comment by armancohan** [comment]
Yes, please see https://github.com/allenai/longformer/issues/181 for additional information.
